docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -176,3 +177,30 @@ For the remaining users:
 
 .. literalinclude:: resource-groups-example.json
     :language: json
+
+.. _scheduleweight-ex:
+
+``schedulingWeight`` example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Schedule weighting is a method of assigning a priority to a resource. Higher
+priority resources are assigned a higher scheduling weight in order to ensure
+that they are run in a more timely manner.
+
+For example, if you had three subgroups under global defined like the
+example below you would end up with this type of distribution:
+
+.. code-block:: text
+
+    data_definition - schedulingWeight: 1 - share: 8.3% (1/12)
+    adhoc - schedulingWeight: 10 - share: 83.3% (10/12)
+    pipeline - schedulingWeight: 1 - share: 8.3% (1/12)
+
+The weight is essentially a ratio. The following example would have the same net
```

Comment:
This phrasing is odd, because a single weight doesn't make a ratio. Would suggest something like "The ratio between scheduling weights determines scheduling priority across subgroups. Because the following example maintains the same ratio between scheduling weights, the net effect is the same."